src/components/draggable-tree/tree.js

```diff
@@ -166,23 +166,24 @@ export const Tree = (props) => {
     }
 
     const renderTreeNode = (node) => {
+        const nodeName = `${node[0]}${node[0] !== 'tempZone' ? `(${node[1]})` : ''}`
```

Comment:
i'm concerned about the hard-coded 'tempZone'... is that... good?

Reply:
No, it's not good. It should come from the zone report. Forgot that :)

Reply:
sold.